fabric-transitive-access-wideners-v1/src/main/resources/fabric-transitive-access-wideners-v1.accesswidener

```diff
@@ -82,6 +82,9 @@ transitive-accessible class net/minecraft/client/model/ModelPart$Quad
 # Creating custom render layers
 transitive-accessible method net/minecraft/client/render/RenderLayer of (Lnet/minecraft/client/render/RenderPhase$Shader;)Lnet/minecraft/client/render/RenderLayer$MultiPhaseParameters;
 
+# Registering custom block entity renderers
```

Comment:
This should be done by modifying the template file instead.

Reply:
Fixed thanks I didn't know that was a issue

Reply:
It's done this way because some of them are auto-generated. Did you try rerunning the generation task to ensure that it matches the edits to the template?

Reply:
Yes it matches